BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Models/AttributeModel.cs

```diff
@@ -1,38 +1,42 @@
 ﻿namespace BugsnagUnityPerformance
 {
 
-    public class AttributeModel
+   public class AttributeModel
```

Comment:
nit: formatting (3 spaces)?
```suggestion
    public class AttributeModel
```